spec/polymorphism_spec.cr

```diff
@@ -0,0 +1,64 @@
+require "./spec_helper.cr"
+
+private class CommentForm < Comment::SaveOperation
+  permit_columns commentable_id
+  permit_columns commentable_type
+
+  def prepare
+    # Make sure we only save values we understand
+    validate_inclusion_of commentable_type, in: ["Employee", "Company"]
+  end
+end
+
+describe "Avram::Polymorphism" do
+  it "allows to create (and return) polymorphic relationships" do
+    post = PostBox.create
+    company = CompanyBox.create
+    employee = EmployeeBox.create
+
+    comment1 = CommentBox.create &.post_id(post.id).body("Comment 1")
+    CommentForm.update!(comment1, {"commentable_type" => "Company", "commentable_id" => company.id.to_s})
+
+    comment2 = CommentBox.create &.post_id(post.id).body("Comment 2")
+    CommentForm.update!(comment2, {"commentable_type" => "Company", "commentable_id" => company.id.to_s})
+
+    comment3 = CommentBox.create &.post_id(post.id).body("Comment 3")
+    CommentForm.update!(comment3, {"commentable_type" => "Employee", "commentable_id" => employee.id.to_s})
```

Comment:
Rather than a hash you can pass named args, for example `commentable_type: "Employee"`

Reply:
solved, thanks for the tip heheh you told me this before, but my memory usually fails hahaha